src/hb-subset-plan.cc

```diff
@@ -324,6 +322,8 @@ _populate_unicodes_to_retain (const hb_set_t *unicodes,
   }
   else
   {
+    // This approach is slower, but can handle adding in glyphs to the subset and will match
+    // them with cmap entries.
     hb_map_t unicode_glyphid_map;
     cmap.collect_mapping (hb_set_get_empty (), &unicode_glyphid_map);
     plan->unicode_to_new_gid_list.alloc (unicode_glyphid_map.get_population ());
```

Comment:
This line is not good... Not from this change, but I just spotted it...

Reply:
Maybe resize by population of `unicodes` plus that of `glyphs`.

Reply:
Makes sense

Reply:
Done.